tokenomics-explainers/001-why-rework.md

```diff
@@ -38,7 +38,7 @@ Unfortunately, over the last few years, we've seen that these ideals do not play
 
 ### Running minipools is primarily Speculative
 
-Unfortunately, the staking requirement in **Ideal 1** makes the financial success of running minipools overwhelmingly dominated by RPL performance.  
```

Comment:
Wording was awkward to finish previous paragraph with "Unfortunately" and then start the next with "Unfortunately"